### DISCUSSION TITLE
Citation File Format and Zenodo

### DISCUSSION THREAD

**Discussion by bobturneruk** [pykale/pykale#193] [Ideas]
I think this would be a good idea to add:

https://citation-file-format.github.io/

**Comment by haipinglu** [comment]
With 8 authors, this will take dozens of lines, not as compact as the current simple bib style. Moreover, it is inconvenient for others to cite in bib style in latex. Our current format is good, which can be copied directly when others write their papers citing PyKale.

**Reply by bobturneruk** [reply]
Thanks @haipinglu! It is possible to output the format as bib - see the "Cite this repository" button on the right here: https://github.com/citation-file-format/ruby-cff (sorry - this was kinda the new thing and not covered in the original link, my fault!). 

Also, CFF should make pykale more findable and accessible as it is machine and human readable. It also enables links to orcid which connects to author metadata. This hopefully translates to reaching more users and improves pykale as an open research output.

Maybe worth considering again? Sorry I didn't explain any of this before!

**Reply by haipinglu** [reply]
Thank you @bobturneruk. Yes, the benefits are clear now. That's why interpretation/explanation is important 👍.
Let's talk when we meet next time.

**Reply by haipinglu** [reply]
Example from MONAI: https://github.com/Project-MONAI/MONAI/blob/dev/CITATION.cff

**Comment by haipinglu** [comment]
@Mdnaimulislam Another enhancement that can be incorporated to pykale, which can be done together with Zenodo.

**Reply by Mdnaimulislam** [reply]
I was just researching How to Make GitHub Repositories Citable Using Zenodo.  I will look more deeply into it.

**Comment by haipinglu** [comment]
Zenodo https://zenodo.org/record/7013889#.Y0AAx3bMLAk: 

I removed .zenodo.json in https://github.com/pykale/pykale/pull/340/, which does not seem to be how MONAI/sklearn is doing. Learn from them if we want to add back. https://zenodo.org/record/6968622#.Yv318nbMKp4  https://zenodo.org/record/6903385#.Yv35Q3bMKp4 or leave it if not worthy.

Mark this as answered when we can make a decision @Mdnaimulislam.
